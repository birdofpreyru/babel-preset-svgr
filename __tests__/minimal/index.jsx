/**
 * Tests the preset used without any options.
 */

import ReactDOM from 'react-dom/server';

import Logo from './logo.svg';

test('SVG import works as expected', () => {
  const code = ReactDOM.renderToString(<Logo />);
  expect(code).toMatchSnapshot();
});
