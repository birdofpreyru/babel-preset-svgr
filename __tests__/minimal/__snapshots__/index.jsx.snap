// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SVG import works as expected 1`] = `"<svg xmlns=\\"http://www.w3.org/2000/svg\\" width=\\"150.517\\" height=\\"188.611\\" viewBox=\\"0 0 39.824 49.903\\" data-reactroot=\\"\\"><path style=\\"opacity:1;fill:#f5f5f5;fill-opacity:1;stroke:none;stroke-width:0.26499999;stroke-linecap:round;stroke-linejoin:miter;stroke-miterlimit:4;stroke-dasharray:none;stroke-dashoffset:0;stroke-opacity:1\\" d=\\"M43.52 87.656h39.824v49.903H43.52z\\" transform=\\"translate(-43.52 -87.656)\\"></path><g aria-label=\\"P\\" style=\\"font-style:normal;font-variant:normal;font-weight:400;font-stretch:normal;font-size:99.75062561px;line-height:120.00000477%;font-family:Oswald;-inkscape-font-specification:Oswald;letter-spacing:0;word-spacing:0;fill:#313131;fill-opacity:1;stroke:none;stroke-width:0.39862508;stroke-linecap:butt;stroke-linejoin:miter;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1\\"><path style=\\"fill:#313131;fill-opacity:1;stroke:none;stroke-width:0.39862508;stroke-miterlimit:4;stroke-dasharray:none\\" d=\\"M64.837 113.732h8.878c6.45 0 11.371-1.829 14.763-5.486 3.391-3.658 5.087-8.778 5.087-15.362 0-7.315-1.596-12.868-4.788-16.658-3.126-3.79-8.18-5.686-15.162-5.686h-20.05l-.053 51.254h-9.947v-60h60v60H64.827Z\\" transform=\\"matrix(.66374 0 0 .66374 -28.916 -41.015)\\"></path><path style=\\"fill:#313131;fill-opacity:1;stroke:none;stroke-width:0.39862508;stroke-miterlimit:4;stroke-dasharray:none\\" d=\\"M70.024 105.652c3.325 0 5.885-.365 7.68-1.097 1.863-.731 3.193-1.995 3.99-3.79.799-1.796 1.198-4.39 1.198-7.781 0-4.123-.3-7.149-.898-9.077-.599-1.929-1.796-3.292-3.591-4.09-1.73-.798-4.489-1.197-8.28-1.197h-5.286v27.032z\\" transform=\\"matrix(.66374 0 0 .66374 -28.916 -41.015)\\"></path></g><text xml:space=\\"preserve\\" style=\\"font-style:normal;font-variant:normal;font-weight:400;font-stretch:normal;font-size:6.06339598px;line-height:120.00000477%;font-family:Oswald;-inkscape-font-specification:Oswald;letter-spacing:0;word-spacing:0;fill:#313131;fill-opacity:1;stroke:none;stroke-width:0.26458332;stroke-linecap:butt;stroke-linejoin:miter;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1\\" x=\\"43.138\\" y=\\"136.474\\" transform=\\"translate(-43.52 -87.656)\\"><tspan x=\\"43.138\\" y=\\"136.474\\" style=\\"fill:#313131;fill-opacity:1;stroke:none;stroke-width:0.26458332;stroke-miterlimit:4;stroke-dasharray:none\\">Dr. Pogodin Studio</tspan></text></svg>"`;
